source/docs/software/wpilib-tools/smartdashboard/verifying-smartdashboard-is-working.rst

```diff
@@ -0,0 +1,55 @@
+Verifying SmartDashboard is working
```

Comment:
@sciencewhiz Should this article remark on the SmartDashboard connection indicator as well?

Reply:
yes

Reply:
There's a section on adding a connection indicator here: https://docs.wpilib.org/en/stable/docs/software/wpilib-tools/smartdashboard/smartdashboard-intro.html#adding-a-connection-indicator

It's also worth talking about the connection indication in the titlebar.

Reply:
> It's also worth talking about the connection indication in the titlebar.

I'm not sure what this is. Could you describe it so I can add it to the article?

Reply:
![image](https://user-images.githubusercontent.com/4885091/92431259-e67cc800-f14b-11ea-941e-cf6b551c34d3.png)
![image](https://user-images.githubusercontent.com/4885091/92431285-fb595b80-f14b-11ea-89de-1efd4d16c829.png)

Reply:
Ah, very helpful. Thanks!

Reply:
Added in 95a0d83.